core/src/CMakeLists.txt

```diff
@@ -1,215 +1,245 @@
-KOKKOS_INCLUDE_DIRECTORIES(
-  ${CMAKE_CURRENT_BINARY_DIR}
-  ${CMAKE_CURRENT_SOURCE_DIR}
+kokkos_include_directories(
+  ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR}
   ${KOKKOS_TOP_BUILD_DIR}
 )
-IF (NOT desul_FOUND)
-  IF(KOKKOS_ENABLE_CUDA)
-    SET(DESUL_ATOMICS_ENABLE_CUDA ON)
-  ENDIF()
-  IF(KOKKOS_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE)
-    SET(DESUL_ATOMICS_ENABLE_CUDA_SEPARABLE_COMPILATION ON)
-  ENDIF()
-  IF(KOKKOS_ENABLE_HIP)
-    SET(DESUL_ATOMICS_ENABLE_HIP ON)
-  ENDIF()
-  IF(KOKKOS_ENABLE_HIP_RELOCATABLE_DEVICE_CODE)
-    SET(DESUL_ATOMICS_ENABLE_HIP_SEPARABLE_COMPILATION ON)
-  ENDIF()
-  IF(KOKKOS_ENABLE_SYCL)
-    SET(DESUL_ATOMICS_ENABLE_SYCL ON)
-    IF(KOKKOS_IMPL_SYCL_DEVICE_GLOBAL_SUPPORTED AND NOT KOKKOS_IMPL_HAVE_SYCL_EXT_ONEAPI_DEVICE_GLOBAL)
-      SET(DESUL_ATOMICS_ENABLE_SYCL_SEPARABLE_COMPILATION ON)
-    ENDIF()
-  ENDIF()
-  IF(KOKKOS_ENABLE_OPENMPTARGET)
-    SET(DESUL_ATOMICS_ENABLE_OPENMP ON) # not a typo Kokkos OpenMPTarget -> Desul OpenMP
-  ENDIF()
-  IF(KOKKOS_ENABLE_OPENACC)
+if(NOT desul_FOUND)
+  if(KOKKOS_ENABLE_CUDA)
+    set(DESUL_ATOMICS_ENABLE_CUDA ON)
+  endif()
+  if(KOKKOS_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE)
+    set(DESUL_ATOMICS_ENABLE_CUDA_SEPARABLE_COMPILATION ON)
+  endif()
+  if(KOKKOS_ENABLE_HIP)
+    set(DESUL_ATOMICS_ENABLE_HIP ON)
+  endif()
+  if(KOKKOS_ENABLE_HIP_RELOCATABLE_DEVICE_CODE)
+    set(DESUL_ATOMICS_ENABLE_HIP_SEPARABLE_COMPILATION ON)
+  endif()
+  if(KOKKOS_ENABLE_SYCL)
+    set(DESUL_ATOMICS_ENABLE_SYCL ON)
+    if(KOKKOS_IMPL_SYCL_DEVICE_GLOBAL_SUPPORTED
+       AND NOT KOKKOS_IMPL_HAVE_SYCL_EXT_ONEAPI_DEVICE_GLOBAL
+    )
+      set(DESUL_ATOMICS_ENABLE_SYCL_SEPARABLE_COMPILATION ON)
+    endif()
+  endif()
+  if(KOKKOS_ENABLE_OPENMPTARGET)
+    set(DESUL_ATOMICS_ENABLE_OPENMP ON
+    )# not a typo Kokkos OpenMPTarget -> Desul OpenMP
```

Comment:
Does the format reject the space between the closing paren and the comment?
Is that something that can be changed?

Reply:
I think trailing comments are considered part of the command. Not sure what happened here but we can just move the comment to the line above.